include/robot_localization/filter_base.h

```diff
@@ -385,6 +388,15 @@ class FilterBase
     //!
     void setSaveRejectedMeasurementTopics(bool save);
 
+    //! @brief Returns a copy of any rejected measurements topics and clears the internal vector
```

Comment:
Looks like this comment block was copied from above. Can you please update it for this new function?